### ISSUE TITLE
Unexpected crash when backup volume is missing.

### ISSUE THREAD

**Issue by TH3L0N3WOLF** [digint/btrbk#19]
I'm sure I'm missing something obvious, but I've tried everything and haven't been able to figure it out.

Running btrbk with the backup volume connected works as expected. 
Disconnecting the backup volume and running again, I get this output:
```
WARNING: Skipping target "/mnt/backups/Laptop/btrbk": Failed to fetch subvolume detail
WARNING: Skipping target "/mnt/backups/Laptop/btrbk": Failed to fetch subvolume detail
WARNING: Skipping target "/mnt/backups/Laptop/btrbk": Failed to fetch subvolume detail
WARNING: Skipping target "/mnt/backups/Laptop/btrbk": Failed to fetch subvolume detail

ERROR: process died unexpectedly (btrbk v0.17.0)
Please contact the author: Axel Burri <axel@tty0.ch>

Stack Trace:
----------------------------------------
Died at /usr/bin/btrbk line 1725, <FILE> line 97.
 at /usr/bin/btrbk line 103, <FILE> line 97.
	main::__ANON__("Died at /usr/bin/btrbk line 1725, <FILE> line 97.\x{a}") called at /usr/bin/btrbk line 1725
```

My config file is as follows:

```
snapshot_dir               .snapshots
incremental                yes
snapshot_create_always     yes
resume_missing             yes
preserve_day_of_week       sunday
snapshot_preserve_daily    14
snapshot_preserve_weekly   4
snapshot_preserve_monthly  0
target_preserve_daily      40
target_preserve_weekly     32
target_preserve_monthly    all
btrfs_commit_delete        after
btrfs_progs_compat         no

volume /mnt/subvolroot
  subvolume rtfs
    target send-receive /mnt/backups/Laptop/btrbk

  subvolume boot
    target send-receive /mnt/backups/Laptop/btrbk

  subvolume home
    target send-receive /mnt/backups/Laptop/btrbk

  subvolume etc
    target send-receive /mnt/backups/Laptop/btrbk

```

I first noticed the issue with version 0.17.0.

**Comment by digint** [comment]
Thanks for the bug report.

Before snapshot creation, btrbk checks the subvolume names on the source AND target, in order not to generate duplicate names (i.e. append correct postfix "_1", "_2" etc if a snapshot or target subvolume already exists with the same date).
If the target is not present, btrbk cannot do this check and cannot create a *verified globally unique* name. If `snapshot_create_always` is set, btrbk will then have to create the snapshot anyways, without knowledge of the target subvolume names. This is no problem as long as btrbk is the only tool managing the source snapshot directory and there is no manual interaction (snapshot delete).

I'm about to fix this, and will also add a note to the documentation like "don't mess with snapshot folders if you enable `snapshot_create_always`"

**Comment by digint** [comment]
Fixed in d28ed9706564a8fcd12dc2d501b39faef2378a98
Fix included in v0.17.1
